configureWebpack.js

```diff
@@ -584,6 +584,10 @@ async function configureWebpack(env) {
                 inject: true,
                 mode: 'webapp',
                 favicons: {
+                    appName: appName,
+                    theme_color: '#000',
+                    appleStatusBarStyle: 'black',
+                    start_url: '/mobile',
```

Comment:
Do we want to assume that most apps will use this url for their mobile app? The "Add to Homescreen" option sets the url to domain + start_url, and does not allow the user to change it. Without it, we're relying on nginx or something doing the redirect.

Reply:
Good question - I think we might as well go ahead and make it configurable as a top-level arg to this method, with a default (that we expect to be used for almost all apps) of `mobile`.  

Could call the arg `defaultMobilePath`.  Open to other names - we have `devServerOpenPage` with that term `page` - but that's related to the specific webpack config and seems a bit weird to use here.

Probably less surprising to have the code inside this method prepend the leading `/` so the dev doesn't need to.